src/workerd/api/basics.c++

```diff
@@ -820,6 +820,9 @@ void AbortSignal::triggerAbort(
 void AbortSignal::serialize(jsg::Lock& js, jsg::Serializer& serializer) {
   auto& handler = JSG_REQUIRE_NONNULL(serializer.getExternalHandler(), DOMDataCloneError,
       "AbortSignal can only be serialized for RPC.");
+
+  LOG_PERIODICALLY(WARNING, "NOSENTRY ", "An AbortSignal was serialized for RPC");
```

Comment:
I'm not aware of any reason to split the string here. It's probably harmless but it could result in weird formatting, I'm not sure, so I'd not do that.

```suggestion
  LOG_PERIODICALLY(WARNING, "NOSENTRY An AbortSignal was serialized for RPC");
```